ynput/ayon-core
# Move houdini client code

## Changelog Description
Move Houdini client code.. 
This is one more step towards splitting addons to their repos.



## Additional info
- Houdini addon in this PR works in dev mode. you'd need to set the addon directory to 
  ```
  <your-ayon-core-clone>/server_addon/houdini/client
  ```
- To create addon zip file, you can do any one of the following
- Run `python server_addon\create_ayon_addons.py --addon houdini`
- Copy `create_package.py` from https://github.com/ynput/example-studio-addon/pull/4 to `<your-ayon-core-clone>/server_addon/houdini` and Run it.


## Additional info 2
I don't think that settings are applied correctly in the publish plugins.
Any advice are much appreciated.

## Testing notes:
1. Update the Houdini addon on server. 
2. Launch Houdini via ayon launcher
3. Everything should work as usual 
   - Create / Publish
   - Manage / Load
   - Work Files tool
   - etc
